src/OrchardCore.Modules/OrchardCore.Users/Controllers/AdminController.cs

```diff
@@ -526,13 +526,19 @@ public async Task<IActionResult> EditPassword(ResetPasswordViewModel model)
             return Forbid();
         }
 
+        // Exclude validating the current password while changing the current user password
+        if (user.UserName != User.Identity.Name)
+        {
+            ModelState.Remove(nameof(EditPasswordViewModel.CurrentPassword));
+        }
+
         if (ModelState.IsValid)
         {
             var token = await _userManager.GeneratePasswordResetTokenAsync(user);
 
             if (await _userService.ResetPasswordAsync(model.UsernameOrEmail, token, model.NewPassword, ModelState.AddModelError))
             {
-                await _notifier.SuccessAsync(H["Password updated correctly."]);
+                await _notifier.SuccessAsync(H["Your password has been changed successfully."]);
```

Comment:
It's not always "Your password". It could be someone else's password. 
```suggestion
                await _notifier.SuccessAsync(H["The user's password has been changed successfully."]);
```

Reply:
Ya it could be